src/garage/tf/algos/npo.py

```diff
@@ -102,6 +102,8 @@ def __init__(self,
         self.policy = policy
         self._scope = scope
         self.max_episode_length = env_spec.max_episode_length
+        if self.max_episode_length == None:
```

Comment:
please place this validation above the long list of property initializations (i.e. first lines of this method), and put a newline after the if statement